docs/FlaskSQLAlchemy.md

```diff
@@ -0,0 +1,419 @@
+* Builds on top of the core- uses the Core to generate SQL and talk to the database.
+
+**Note:** For the complete guide, checkout the API documentation on the [SQLAlchemy](https://flask-sqlalchemy.palletsprojects.com/en/2.x/api/#flask_sqlalchemy.SQLAlchemy) class.
+
+### Requirements
+
+| Our Version             | Python                  | Flask                   | SQLAlchemy              |
+|-------------------------|-------------------------|-------------------------|-------------------------|
+| 2.x                     | 2.7, 3.4+               | 0.12+                   | 0.8+ or 1.0.10+ w/      |
+|                         |                         |                         | Python 3.7              |
+| 3.0+ (in dev)           | 2.7, 3.5+               | 1.0+                    | 1.0+                    |
+
+### Prerequisites
+
+* Install Python using PPA (Personal Package Archive)
+
+  * Install the following requirements:
+    ```
+    sudo apt-get install software-properties-common python-software-properties
+    ```
+  * Run the following command to add the PPA
+    ```
+    sudo add-apt-repository ppa:deadsnakes/ppa
+    ```
+  * Update
+    ```
+    sudo add-get update
+    ```
+  * Install Python 3.6
+    ```
+    sudo apt-get install python3.6
+    ```
+  * You can check the installed version by using this command
+    ```
+    python3.6 --version
+    ```
+
+* Check if pip (The python package installer) is already installed by running
+  ```
+  pip --version
+  ```
+  * If installed then upgrade it using the command
+    ```
+    python -m pip install --upgrade pip
+    ```
+    **Note:** This will upgrade the pip version
+
+
+* Install virtual environment
+  ```
+  pip install virtualenv
+  ```
+  * Create python3.6 virtual environment
+    ```
+    virtualenv -p python3.6 venv
+    ```
+  * Activate virtual environment
+    ```
+    source venv/bin/activate
+    ```
+    
+* Install Flask (version 1.0.2)
+  ```
+  pip install Flask==1.0.2
+  ```
+
+* Install Flask-SQLAlchemy (version 2.3.2)
+  ```
+  pip install Flask-SQLAlchemy==2.3.2
+  ```
+  **Note:** You can use any version of **Python**, **Flask** & **Flask-SQLAlchemy** depending on the **Requirements** as stated above
+
+### Create an Application
+
+To create your Flask application, load the [configuration](https://flask-sqlalchemy.palletsprojects.com/en/2.x/config/) of choice and then create the [SQLAlchemy](https://flask-sqlalchemy.palletsprojects.com/en/2.x/api/#flask_sqlalchemy.SQLAlchemy) object by passing it the application.
+Once created, that object then contains all the functions and helpers from both sqlalchemy and sqlalchemy.orm. Furthermore it provides a class called Model that is a declarative base which can be used to declare models:
+
+```
+    from flask import Flask
+    from flask_sqlalchemy import SQLAlchemy
+
+    app = Flask(__name__)
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:////tmp/test.db'
```

Comment:
There are 4 slashes still. Did you miss something or is it this way?

Reply:
Yes it is that way only for MAC/UNIX 4 slashes are required